specification/cognitiveservices/data-plane/Personalizer/preview/v1.0/Personalizer.json

```diff
@@ -489,6 +489,7 @@
           "200": {
             "description": "Success",
             "schema": {
+              "$ref": "#/definitions/Stream",
               "type": "file"
```

Comment:
Not sure if we support `"file"` type here. Since the return type is an object model, please mark the type as `object`